### PR TITLE
feat(swift): update parser and queries to 0.3.0

### PR DESCRIPTION
**Pull request by clason**
I see that our queries are almost-but-not-completely the same as the upstream ones. Is the divergence intentional, or should/can I just pull in the latest upstream version?

@alex-pinkus @theHamsta

### CODE REVIEW

**Comment by clason** [comment]
...which I just did, as a separate commit to be squashed so you can look at the non-necessary changes.

**Comment by alex-pinkus** [comment]
I think those are from pull requests that happened upstream but never got incorporated here. E.g. https://github.com/alex-pinkus/tree-sitter-swift/pull/135 — should be safe to pull in here

**Comment by clason** [comment]
Thanks! Having a versioned upstream that provides usable queries makes updates easy. 

(It would be good to document this somewhere -- which parsers use upstream queries and which don't, so we don't duplicate efforts and can prevent divergence by pointing PRs to them upstream.)

File: queries/swift/highlights.scm [hunk 1]
```diff
@@ -3,7 +3,6 @@
 
 ; Identifiers
 (attribute) @variable
-(simple_identifier) @variable
```
Comment (theHamsta) [inline queries/swift/highlights.scm:6]: Any reason to remove this?

**Review by theHamsta — APPROVED** [review]

File: queries/swift/highlights.scm (hunk 1 above)
Comment (clason) [reply queries/swift/highlights.scm:6]: To reflect upstream changes; I believe it's replaced by 
```
(pattern bound_identifier: (simple_identifier)) @variable
```
further down.

Comment (alex-pinkus) [reply queries/swift/highlights.scm:6]: In https://github.com/alex-pinkus/tree-sitter-swift/pull/135, the PR author complained that it was “too broad, it overlaps with many other patterns.” I’m not sure why that was causing a problem but it seemed fine to me to narrow it.

**Pull request merged**